test/tests/times/tests

```diff
@@ -0,0 +1,47 @@
+[Tests]
+  issues = '#24696'
+  design = 'syntax/Times/index.md'
+
+  [test]
+    requirement = "The system shall be able to gather times from"
+
+    [input]
+      type = 'JSONDiff'
+      input = 'input_times.i'
+      jsondiff = 'input_times_out.json'
+      detail = 'a parameter in the input file,'
+    []
+    [functors]
+      type = 'JSONDiff'
+      input = 'functor_times.i'
+      jsondiff = 'functor_times_out.json'
+      detail = 'a functor evaluated at the element containing the origin,'
+      # Needs to be able to find the element containing the origin
```

Comment:
Ah this answered my question from before ... point locator will only be globally consistent on replicated mesh